packages/open-next/src/adapters/cache.ts

```diff
@@ -192,7 +193,11 @@ export default class S3Cache {
       } as CacheHandlerValue;
     } catch (e) {
       // We can usually ignore errors here as they are usually due to cache not being found
-      debug("Failed to get fetch cache", e);
+      if (isIgnorableError(e)) {
```

Comment:
Do we want to do this here, `error` should already be handling this https://github.com/opennextjs/opennextjs-aws/blob/4a500eeb609e1ed3be97d744e5d762d65dafd883/packages/open-next/src/adapters/logger.ts#L43-L67
Having only the `error` function deciding which error to log is probably better (and easier to read)

Reply:
I think `IgnorableError` alone are enough in most cases, debug mode should never be used outside of when you debug so you should never see such errors in production.
The `OPEN_NEXT_ERROR_LOG_LEVEL` has 1 main use imo, it's for adding the ability to mask `RecoverableError` logs, but it could also be used to mask `IgnorableError` in debug mode, which could be useful for us when testing something that has nothing to do with these errors